Update sql traits frequency settings (#2146)
segmentio/segment-docs: src/personas/sql-traits.md

![](images/sql_traits_connect3.png)
Check **Compute without destinations** if you only want to send to Personas

> When you create a SQL Trait, Segment runs the query on the warehouse twice a day by default. You can customize what time Segment will query the data warehouse as well as the frequency, up to once per hour, from the SQL Trait's settings.

Reviewer: ```suggestion
When you create a SQL Trait, Segment runs the query on the warehouse twice a day by default. You can customize the time at which Segment queries the data warehouse and  the frequency, up to once per hour, from the SQL Trait's settings.
```
